.github/workflows/ci.yaml

```diff
@@ -0,0 +1,270 @@
+name: CI Tasks for Ory Hydra
```

Comment:
Could you merge the test in https://github.com/ory/hydra/blob/master/.github/workflows/oidc-conformity.yml into this workflow?

Reply:
ps: this test can be run on all branches and PRs :) not sure why it was only master originally

Reply:
Done!